tool/tsh/registry_windows.go

```diff
@@ -0,0 +1,81 @@
+//go:build windows
+
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"errors"
+	"os"
+	"strconv"
+
+	"github.com/gravitational/trace"
+	"golang.org/x/sys/windows/registry"
+)
+
+// getRegistryKey loads or creates a registry key handle and passes it back
```

Comment:
Do you still need this code now that you've moved it to the registry package?

Reply:
No, deleted.